proto/ignition/msgs/conegeom.proto

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+option java_outer_classname = "ConeGeomProtos";
+
+/// \ingroup ignition.msgs
+/// \interface ConeGeom
+/// \brief Information about a cone geometry
+
+import "ignition/msgs/header.proto";
+
+message ConeGeom
+{
+  /// \brief Optional header data
+  Header header  = 1;
+
+  double radius  = 2;
```

Comment:
Could you document all fields in the new message? The unit would also be important (i.e. `meters`). Thanks!

Reply:
Done